specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json

```diff
@@ -1206,10 +1233,11 @@
       ]
     },
     "update_session_settings": {
+      "description": "Settings used to update the session.",
       "type": "object",
       "properties": {
         "maxLeaseTimeMinutes": {
-          "description": "Update to the time the session will run after it reached the 'Ready' state.  It has to be bigger than the current value of maxLeaseTimeMinutes.",
+          "description": "Update to the time the session will run after it reached the 'Ready' state. It has to be bigger than the current value of maxLeaseTimeMinutes and less than 1440.",
```

Comment:
I'm not a native speaker, but usually you say "larger than" or "greater than" instead of "bigger"? 
-> Nit